tests/sentry/api/endpoints/test_organization_projects.py

```diff
@@ -149,3 +149,18 @@ def test_api_key(self):
             self.path, HTTP_AUTHORIZATION="Basic " + b64encode(u"{}:".format(key.key))
         )
         self.check_valid_response(response, [project])
+
+    def test_all_projects(self):
+        self.login_as(user=self.user)
+        other_team = self.create_team(organization=self.org)
+
+        projects = []
+        # Create 101 projects to verify the project list is not paginated
```

Comment:
This is a pretty expensive test for this functionality. Let's not create 100 projects, you can instead just pass a lower page limit via `per_page`, and just create 2-3.

Reply:
ahh yea that makes sense I should've seen that from the previous tests